kk/tests/test_images.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+import datetime
+import urllib
+
+from django.conf import settings
+
+from kk.models import Hearing, HearingImage
+from kk.tests.base import BaseKKDBTest
+
+class TestImage(BaseKKDBTest):
+    def setup(self):
+        super(TestImage, self).setup()
+
+        self.hearing = Hearing(abstract='Hearing One')
```

Comment:
This will only be executed once, and then mutated through the tests, meaning you'll have, say, 3 images on it to begin with, then 6 for another test, then maybe 9... Probably not what you want?